app/assets/javascripts/expanded_textarea.js

```diff
@@ -0,0 +1,7 @@
+// make textareas expand to their contents size
+// adjusting the math (+3) to make 1-line have regular size
+$(function(){
+  $('textarea').each(function(i, element){
+    element.style.height = element.scrollHeight + 3 +"px";
```

Comment:
might be worth using jquery here too..

Reply:
doing it for the speed :D 
... if it somehow fails we can change it over to jquery ...